tests/integration/backward_compatible/serialization/compact_compatibility/compact_compatibility_test.py

```diff
@@ -996,6 +996,13 @@ def test_put(self):
         self.assertTrue(self.multi_map.put(INNER_COMPACT_INSTANCE, OUTER_COMPACT_INSTANCE))
         self.assertEqual([OUTER_COMPACT_INSTANCE], self.multi_map.get(INNER_COMPACT_INSTANCE))
 
+    # def test_put_all_get(self):
+    #     self.multi_map.put_all(
+    #         {"key1": ["value1", "value2", "value3"], "key2": ["value4", "value5", "value6"]}
```

Comment:
You should use some Compact values here, otherwise you are not testing the compatibility with Compact